server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndex.java

```diff
@@ -263,4 +279,9 @@ private static QueryBuilder toValueQuery(MetricCriterion criterion) {
     }
   }
 
+  @FunctionalInterface
+  private interface FacetSetter {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Drop this interface in favor of "java.util.function.BiConsumer<SearchRequestBuilder,Map>". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1711)